example/memfs_ll.cc

```diff
@@ -1087,6 +1087,7 @@ static const struct fuse_lowlevel_ops memfs_oper = {
 	.copy_file_range = nullptr,
 	.lseek = nullptr,
 	.tmpfile = nullptr,
+	.statx = nullptr,
```

Comment:
#ifdef HAVE_STATX ?

Reply:
Btw, interesting that the compiler does not complain for passthrough_hp.cc that it is not set (although I find missing C99 initialization silly in c++, maybe I should update standard for these files to c++20, which has that.